build.gradle

```diff
@@ -10,7 +10,7 @@ repositories {
 	mavenCentral()
 }
 
-def runeLiteVersion = '1.9.3'
+def runeLiteVersion = '1.9.8'
```

Comment:
Lets change this to 
`def runeLiteVersion = 'latest.release'`
So we don't have to change it again next update

Reply:
I've seen this before but for some reason I thought it was a placeholder :facepalm: